pages/map.js

```diff
@@ -56,6 +56,8 @@ export default function Map({ data }) {
   const [filteredUsers, setFilteredUsers] = useState([]);
   const [selectedTags, setSelectedTags] = useState(new Set());
 
+  console.log(selectedTags);
```

Comment:
Console Logs are alright while debugging but best remove them after your done (standard production level project guidlines)

Reply:
Sorry, my bad. It's fixed now.